### PR TITLE
German: Style and period.

### PR DESCRIPTION
**Pull request by DonSqueak**
Needed to rewrite one sentence as it just didn't feel right/capture the right mood. Added a period.

### CODE REVIEW

**Comment by DonSqueak** [comment]
Are these here still on your radar?

**Comment by DonSqueak** [comment]
Also, "no override" is missing:

**Comment by DonSqueak** [comment]
@technocidal Take a look please as well, please, if you can. Note: I changed "Würdigungen" to "Credits" in German yesterday, as I believe that "Credits" entered German as a loan word already and it captures the overall colloquial style of Callsheet better than Würdigungen, which has a bit of an archaic vibe to it, as well as being slightly... highfalutin if you will?

File: CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff [hunk 1]
```diff
@@ -369,7 +369,7 @@
       </trans-unit>
       <trans-unit id="Credits Lists" xml:space="preserve">
         <source>Credits Lists</source>
-        <target state="translated">Credits enthalten …</target>
+        <target state="translated">Auswahl für Credits</target>
```
Comment (DonSqueak) [inline CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff:372]: "Credits enthalten ..." just didn't look and feel right. "Auswahl für Credits" means "Selection for Credits" which sacrifices the noun-verb-structure, but overall feels... slightly better here imho.

**Review by cliss — APPROVED** [review]

**Comment by cliss** [comment]
> Are these here still on your radar?
> 
>  

These should be there already; if not, let me know:

Screenshot 2024-05-14 at 4 03 10 PM

**Comment by cliss** [comment]
Actually re: the IAP stuff, you very well may have translated them, but that has to go into Apple's backend, which I haven't done yet. 🤪

**Comment by DonSqueak** [comment]
> Actually re: the IAP stuff, you very well may have translated them, but that has to go into Apple's backend, which I haven't done yet. 🤪

So, I have indeed translated the stuff you show in the screenshot.
I don't see:
* Yearly Subscription (I do see "Yearly", but Subscription and the Save 17 % part is missing)
* where "per monat" and "per jahr" came from, as this is not in the file (and wrong ;) )

**Comment by cliss** [comment]
Added in 70bece0d3313e756de972eb0f9d1cc12be8e4ea7:

- `Save X%`
- `A few more free ones please?` (shown when on the max-supporter purchasing screen)
- `Free, but you can only ask once. 😏` (subtitle for the above)

That said, I'm 99% sure the `per month` is [coming straight from Apple](https://developer.apple.com/documentation/storekit/product/subscriptionperiod/unit/3916875-localizeddescription).

**Comment by technocidal** [comment]
@DonSqueak Changing Würdigungen to Credits is the right call. I probably should have done it already when I decided that Crew is an established loan word as well and probably better known than Filmstab.

**Pull request merged**